CommunicationServer/Application/Application.csproj

```diff
@@ -10,4 +10,10 @@
         <ProjectReference Include="..\Domain\Domain.csproj"/>
     </ItemGroup>
 
+    <ItemGroup>
+      <Reference Include="System.Reactive">
```

Comment:
That packet referencing weirdly doesnt work on windows machines. Maybe import it the same way we import chessLib

Reply:
sure

Reply:
Thanks, I appreciate it!